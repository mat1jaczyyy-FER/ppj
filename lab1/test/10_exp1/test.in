a+b*(12+43/4)*12-(23*(2*(4)))
