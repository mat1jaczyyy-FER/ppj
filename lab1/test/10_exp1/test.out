IDN 1 a
OP_PLUS 1 +
IDN 1 b
OP_PUTA 1 *
L_ZAGRADA 1 (
BROJ 1 12
OP_PLUS 1 +
BROJ 1 43
OP_DIJELI 1 /
BROJ 1 4
D_ZAGRADA 1 )
OP_PUTA 1 *
BROJ 1 12
OP_MINUS 1 -
L_ZAGRADA 1 (
BROJ 1 23
OP_PUTA 1 *
L_ZAGRADA 1 (
BROJ 1 2
OP_PUTA 1 *
L_ZAGRADA 1 (
BROJ 1 4
D_ZAGRADA 1 )
D_ZAGRADA 1 )
D_ZAGRADA 1 )
